modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/EditAPIApplication.testcase

```diff
@@ -29,11 +29,213 @@ definition {
 		}
 	}
 
+	@priority = 4
```

Comment:
Hi @pedroamarall , thanks for the files. I just checked the LPS automation ticket, this is a new file we would like to add, please
- create a new testcase file named `EditAPIApplicationSchema.testcase` in this existing folder `[UI]DefineAPISchemasWithinAnAPIGroup`